sql/testdata/builtin_function

```diff
@@ -219,3 +219,118 @@ query T
 SELECT right('abcde', 2)
 ----
 de
+
+# TIMESTAMP/DATE builtins.
+
+query T
+SELECT age(timestamp '2001-04-10 22:06:45', timestamp '1957-06-13')
+----
+384190h6m45s
+
+query T
+SELECT age(timestamp '1957-06-13') - age(now(), timestamp '1957-06-13') < interval '1s'
+----
+true
+
+query T
+SELECT now() - timestamp '2015-06-13' > interval '100h'
+----
+true
+
+query T
+SELECT now() - clock_timestamp() < interval '1s' 
+----
+true
+
+query T
+SELECT now() - current_date::timestamp < interval '24h1s'
+----
+true
+
+query T
+SELECT now() - current_timestamp < interval '1s'
```

Comment:
It looks like some of these work with or without parens, while others require parens. We should have a test that `current_timestamp()` works and `now` doesn't.